addons/web/static/src/js/fields/relational_fields.js

```diff
@@ -2286,11 +2286,15 @@ var FieldMany2ManyTags = AbstractField.extend({
         this.many2one._getSearchCreatePopupOptions = function (view, ids, context, dynamicFilters) {
             var options = _getSearchCreatePopupOptions.apply(this, arguments);
             var domain = this.record.getDomain({fieldName: this.name});
+            var m2m_records = []
```

Comment:
missing `;`
also, we camelcase variable names

Reply:
fixed in 56d319b7704f6b495e38539b2019d623f1752c1c